### ISSUE TITLE
Timer in Fractal - Change in time format at the end of a 24hr period

### ISSUE THREAD

**Issue by tomstuart123** [decent-dao/fractal-interface#1402]
A shutter dao member just raised this: 

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/d6ffe7dd-7f06-49fa-ad1d-376aa4c846ed)

Kirill and I noted this in prior conversations 

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/b4c750f4-2de3-497a-8090-fdfdeb338631)

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/38acb375-4888-46b1-b0ee-3b65d77e033c)

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/47e1eb06-996b-40b8-b020-7e0d34a107d7)

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/b754055b-ccc8-4961-9088-cc5d175cd007)

![Image](https://github.com/decent-dao/fractal-interface/assets/31554133/1217a0de-5df9-45b9-b961-b04d1872c666)

In short, as the clock gets to the end of a day remaining it removes the hours label from the clock (e.g. we see a transition FROM 3Days: 1hour: 1 minutes TO 3days:minutes

As this can make it seem like the proposal has hours instead of days to execute, this can be very jarring for the reader and they have to click the proposal and it's end time to confirm all is okay

**Comment by adamgall** [comment]
What I suspect is going on here, is that we're hiding a specific time section when it's "00". So for example, if there are 0 days left, we simply don't show the day section.

But I bet that we're accidentally hiding a section when it's "00", even if a _larger_ time section is _not_ "00". So if there is 1 day, 0 hours, 10 minutes, and 30 seconds left, we should be showing "01:00:10:30", but are accidentally hiding the hours: "01:10:30".

If my assumptions are correct, the fix here is to only hide "00"s when all of the "larger" time sections are also "00".

**Comment by tomstuart123** [comment]
Great. Thanks @adamgall
